lambdas/preview/test/test_index.py

```diff
@@ -27,10 +28,42 @@ def _make_event(cls, query, headers=None):
 
     def test_bad(self):
         """send a known bad event (no input query parameter)"""
-        event = self._make_event({'url': self.FILE_URL})
+        event = self._make_event({'url': self.FILE_URL}, {'origin': MOCK_ORIGIN})
         resp = lambda_handler(event, None)
         assert resp['statusCode'] == 400, "Expected 400 on event without 'input' query param"
         assert resp['body'], 'Expected explanation for 400'
```

Comment:
maybe something dumb about my test setup by pylint vomits all over this file with
```
test_index.py:33:0: E1126: Sequence index is not an int, slice, or instance with __index__ (invalid-sequence-index)
```
probably a bug we should suppress this message?